appcli/orchestrators.py

```diff
@@ -176,6 +176,15 @@ def verify_service_names(
         """
         raise NotImplementedError
 
+    def get_disabled_commands(self) -> list[str]:
+        """
+        Returns the list of commands disabled by this orchestrator.
```

Comment:
Clarify.
```
The list of default appcli commands which this orchestrator wants to disable.
E.g. ['init', 'backup'].

This allows orchestrators to prevent commands being run which are not supported.
```